HelpSource/Classes/PfadeIn.schelp

```diff
@@ -0,0 +1,52 @@
+TITLE::PfadeIn
+SUMMARY::Fade an event pattern in
+CATEGORIES::Streams-Patterns-Events>Patterns>Filter
+RELATED::Classes/PfadeOut
+
+DESCRIPTION::
+Fade an event pattern in by scaling it's amplitude over a period of time.
```

Comment:
should be: "scaling its amplitude"

Reply:
https://github.com/supercollider/supercollider/pull/5841